### ISSUE TITLE
Crystal 0.29 support

### ISSUE THREAD

**Issue by jwoertink** [philnash/crotp#9]
Not sure what's up here, but I'm getting a no method overload error:

```
in lib/crotp/src/crotp/hotp.cr:9: instantiating 'generate_otp(Int32)'

      generate_otp(counter)
      ^~~~~~~~~~~~

in lib/crotp/src/crotp/otp.cr:26: no overload matches 'OpenSSL::HMAC.digest' with types Symbol, String, Slice(UInt8)
Overloads are:
 - OpenSSL::HMAC.digest(algorithm : OpenSSL::Algorithm, key, data)

      digest = OpenSSL::HMAC.digest(algorithm, @secret, bytes.reverse!)
                             ^~~~~~
```

#8 might even fix this, but I just wanted to make sure it was documented.

**Comment by repomaa** [comment]
I'm pretty sure #8 will fix this since it won't be using symbols for passing the algorithm anymore. #8 has been open quite some time, anything needed still for merge?

**Comment by philnash** [comment]
Yup, this changed in Crystal 0.28.0 and #8 has now been merged and released as version 0.3.0. Sorry for the delay with this!
